pages/form-builder/responses/[[...params]].tsx

```diff
@@ -16,15 +16,45 @@ import { useSession } from "next-auth/react";
 import Head from "next/head";
 import { checkOne } from "@lib/cache/flags";
 import Link from "next/link";
+import { Card } from "@components/globals/card/Card";
+import { useRouter } from "next/router";
+import { DownloadTable } from "@components/form-builder/app/DownloadTable/DownloadTable";
+import { ToastContainer, toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.min.css";
+import { logMessage } from "@lib/logger";
+import axios from "axios";
 
 interface ResponsesProps {
   vaultSubmissions: VaultSubmissionList[];
+  formId: string;
```

Comment:
formId is potentianlly undefined here.

```suggestion
  formId?: string;
```